models.py

```diff
@@ -36,8 +37,17 @@ def to_dict(self):
         return meme_dict
 
 
+class React(enum.Enum):
```

Comment:
These should be changed to string values.

Currently, `React('like')` raises a `ValueError`.

Reply:
So just not use enums, and store strings in the database?

Reply:
You can still use enums, but the React enum values should be strings instead of integers.
i.e.
```
class React(enum.Enum):
    like = 'like'
    laugh = 'laugh'
    sad = 'sad'
    angry = 'angry'
    wow = 'wow'
```
Right now, your code won't work because the `React` constructor will never match any of the string React types. Changing it to look like what I just described will allow the constructor to accept strings.

Another thought: When we parse the request body for the react type, we should also normalize it to lowercase.